benchmarks/bls_signature.nim

```diff
@@ -81,9 +81,80 @@ proc benchFastAggregateVerify*(numKeys, iters: int) =
 
   bench("BLS agg verif of 1 msg by " & $numKeys & " pubkeys", iters):
     let valid = validators.fastAggregateVerify(msg, finalSig)
-    doAssert valid
+    # doAssert valid
```

Comment:
Seems easy enough to re-add this if/when needed?

Reply:
That was mostly because I wanted to check if i didn't mess up the internals when testing. I don't think it would change the benchmark numbers.

Reply:
Sure, I'm referring specifically to having code in the PR that's commented out. It's not the existence of the `let valid = ...` line, but that the `doAssert valid` isn't enabled. It's that it'd be easy to re-add the `doAssert valid` line if desired.